Xamarin.Forms.Core/NavigationProxy.cs

```diff
@@ -1,17 +1,19 @@
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
-	internal class NavigationProxy : INavigation
+	public class NavigationProxy : INavigation
 	{
 		INavigation _inner;
 		Lazy<List<Page>> _modalStack = new Lazy<List<Page>>(() => new List<Page>());
 
 		Lazy<List<Page>> _pushStack = new Lazy<List<Page>>(() => new List<Page>());
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Good point. On the subsequent Internals scrub I'll check to see if adding the attribute to the class simply hides all the members.